jerry-core/ecma/base/ecma-helpers-number.c

```diff
@@ -765,7 +765,12 @@ ecma_number_parse_int (const lit_utf8_byte_t *string_buff, /**< routine's first
 
   /* 6. */
   ecma_number_t radix_num;
-  radix = ecma_op_to_numeric (radix, &radix_num, ECMA_TO_NUMERIC_NO_OPTS);
+  radix = ecma_op_to_number (radix, &radix_num);
```

Comment:
The result of this is checked twice, on line 770 and on line 775. We should apply @rerobika's suggestion:
> ```c
> if (ECMA_IS_VALUE_ERROR (ecma_op_to_number (radix, &radix_num)))
> {
>   return ECMA_VALUER_ERROR;
> }
> ```